app/routes/application.js

```diff
@@ -10,7 +10,25 @@ function createNewApplication (req) {
 
   if (typeof data.applications[code] === 'undefined') {
     data.applications[code] = {
-      status: 'started'
+      status: 'started',
```

Comment:
It seems to me that you've added these defaults so that later, in the routes, you can check to see if the user added choices via the following code:

`if (Object.keys(choices).length === 0)`

I don't think we should define empty objects here. Instead I'd make the check:

`if(application.choices)`

If the fact it's empty is a little more complicated (which I think it might be judging from the fact that some properties are set to an empty object and some are set to null) then I'd suggest a method to encapsulate this so I can do:

`if(app.isTaskComplete('choices'))`

Behind the scenes ,this function might hand off to `app.isChoicesComplete()`. 

`

Reply:
Created a new utility function `hasCompletedSection(key)`, which checks for either `null` or empty keys.